package com.jpmc.midascore.foundation;

public class Transaction {
    private long senderId;
    private long recipientId;
    private double amount;

    public Transaction() { }

    public Transaction(long senderId, long recipientId, double amount) {
        this.senderId = senderId;
        this.recipientId = recipientId;
        this.amount = amount;
    }

    public long getSenderId() {
        return senderId;
    }

    public void setSenderId(long senderId) {
        this.senderId = senderId;
    }

    public long getRecipientId() {
        return recipientId;
    }

    public void setRecipientId(long recipientId) {
        this.recipientId = recipientId;
    }

    public double getAmount() {
        return amount;
    }

    public void setAmount(double amount) {
        this.amount = amount;
    }

    @Override
    public String toString() {
        return "Transaction {senderId=" + senderId
                + ", recipientId=" + recipientId
                + ", amount=" + amount + "}";
    }
}
